library/src/main/java/com/novoda/downloadmanager/InternalMigrationStatus.java

```diff
@@ -1,17 +0,0 @@
-package com.novoda.downloadmanager;
-
-interface InternalMigrationStatus extends MigrationStatus {
```

Comment:
Clients have full control of the migration process now except passing to the `download-manager` for storage or download which will go through the normal notification system.